### PR TITLE
Run make codespell

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```
[NO NEW TESTS NEEDED]

### CODE REVIEW

File: cmd/podman/containers/export.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func export(cmd *cobra.Command, args []string) error {
 		if err := parse.ValidateFileName(outputFile); err != nil {
 			return err
 		}
-		// open file here with WRONLY since on MacOS it can fail to open /dev/stderr in read mode for example
+		// open file here with WRONGLY since on MacOS it can fail to open /dev/stderr in read mode for example
```
Comment (Luap99) [inline cmd/podman/containers/export.go:82]: This was correct, the option is called `WRONLY`, alternatively change it to `O_WRONLY`

File: test/e2e/systemd_activate_test.go [hunk 2]
```diff
@@ -72,7 +72,7 @@ var _ = Describe("Systemd activate", func() {
 		defer os.RemoveAll(tmpRootDir)
 
 		// When SELinux is enabled, a storage root directory should be
-		// labeld with a specific value
+		// labelled with a specific value
```
Comment (edsantiago) [inline test/e2e/systemd_activate_test.go:75]: I say, old chap, does this spellchecker suggest `labelled` with double-ell, wot? On this side of the pond, one more traditionally sees the single-el variant, "labeled".

File: test/system/150-login.bats [hunk 3]
```diff
@@ -215,7 +215,7 @@ function _push_search_test() {
                --cert-dir ${PODMAN_LOGIN_WORKDIR}/trusted-registry-cert-dir \
                localhost:${PODMAN_LOGIN_REGISTRY_PORT}/$destname
 
-    # Search a pushed image will be successed
+    # Search a pushed image will be success
```
Comment (edsantiago) [inline test/system/150-login.bats:218]: This one needs a complete rewrite, it doesn't make sense either way.

File: test/e2e/systemd_activate_test.go (hunk 2 above)
Comment (rhatdan) [reply test/e2e/systemd_activate_test.go]: And I thought we revolted against England so we would not have to follow their spelling rules.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Pull request merged**